File: Java/UberX.java
package Java;

public class UberX extends Car {
    String brand;
    String model;
    public UberX(String license, Account driver, String brand, String model) {
        super(license, driver);
        this.brand = brand;
        this.model = model;
        
    }
    @Override
    public void PrintDataCar() {
        super.PrintDataCar();
        System.out.println("Modelo: "+model + "brand: "+brand );
    }
    @Override
    public void setPassenger(Integer passenger) {
        if(passenger == 4){
            this.passenger = passenger;
        }else{
            System.out.println("Necesitas asignar 4 pasajeros");
        }
    }
}

File: PHP/uberX.php
<?php
require_once ('Car.php');
class uberX extends Car{
    public $brand;
    public $model;
    public function __construct($license, $driver, $model, $brand){
        parent::__consturct($license, $driver);
        $this->license = $license;
        $this->driver = $driver;
    }
    public function printDataCar(){
        echo "licencia: $this->license Driver: $this->driver".$this->name;
    }
}
?>